### PR TITLE
Add simple local cluster & hotrod example instructions

### LINKED ISSUE #4: Add setup instructions for how to run the example applications
This will include:

1. Spinning up some local cluster (minikube etc) - this can just be a link out to relevant docs
2. Deploying hotrod - using https://github.com/signadot/hotrod/blob/main/k8s/all-in-one/demo.yaml
3. Connecting cluster to signadot - simple description
4. Using a set of images we have already published to spin up new workspaces (show 2-3 different image tags that we have named appropriately - like `signadot/hotrod-route:bugfix-slow-cars` or `signadot/hotrod-frontend:improve-ui`
 
The goal here is to ensure that someone can come into the application and set up and run these examples to get an understanding of workspaces without having to compile code or build new images.

cc @jmsktm @arjun-sd

### PR DESCRIPTION
**Pull request by Freezerburn**
@foxish Is this basically the kind of thing you were expecting? (I couldn't think of anything to really show off other than the route fix, so if you have specific suggestions for other stuff let me know and I can try to push some images and add that. Also I'm not really sure if the instructions are fully correct, so let me know if there's anything that needs to be fixed)

Closes #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
 # Signadot Examples
 
 This repository contains examples for Signadot users.
+
+# Setup Instructions for Running Examples
+
+1. Ensure you have [Docker](https://www.docker.com/), [minikube](https://minikube.sigs.k8s.io/docs/), [helm](https://helm.sh/), and [kubectl](kubectl apply -f https://raw.githubusercontent.com/signadot/hotrod/main/k8s/all-in-one/demo.yaml) installed.
+2. Run [`minikube start`](https://minikube.sigs.k8s.io/docs/start/) to create a Kubernetes environment locally.
+3. Create our all in one demo application in your local Kubernetes cluster by running `kubectl apply -f https://raw.githubusercontent.com/signadot/hotrod/main/k8s/all-in-one/demo.yaml`.
```
Comment (foxish) [inline README.md:9]: I think this needs namespace to be specified as hotrod.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+4. Create a cluster in the [Signadot Dashboard](https://app.signadot.com/) and make sure to copy the token generated for it for the next step.
+5. Run the following commands:
+    1. `helm repo add signadot https://charts.signadot.com`
+    2. `helm install signadot-workspaces signadot/workspaces`
+    3. `kubectl -n signadot create secret generic cluster-agent --from-literal=token=...` where the `...` is replaced with the token you copied when creating the cluster in the Signadot Dashboard.
+6. Create a new workspace in the Signadot Dashboard in the newly-created cluster. Select "hotrod" as the namespace, and choose "route" in the "Select Deployments to Fork" section.
```
Comment (foxish) [inline README.md:15]: The step 6 and beyond following this are not needed - after this, I'd expect them to just run the integration tests. I was thinking of these 5 steps above to be the pre-requisites section in https://github.com/signadot/examples/blob/main/java-sdk-example/README.md and https://github.com/signadot/examples/blob/main/node-sdk-example/README.md

And then the next step for a user is to run the integration tests.

Comment (Freezerburn) [reply README.md:15]: Gotcha. Your comment about using existing images based on hotrod to spin up new workspaces threw me off here.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
 # Signadot Examples
 
 This repository contains examples for Signadot users.
+
+# Setup Instructions for Running Examples
+
+1. Ensure you have [Docker](https://www.docker.com/), [minikube](https://minikube.sigs.k8s.io/docs/), [helm](https://helm.sh/), and [kubectl](kubectl apply -f https://raw.githubusercontent.com/signadot/hotrod/main/k8s/all-in-one/demo.yaml) installed.
+2. Run [`minikube start`](https://minikube.sigs.k8s.io/docs/start/) to create a Kubernetes environment locally.
```
Comment (foxish) [inline README.md:8]: There's an instruction to create the namespace `hotrod` - `kubectl create ns hotrod`

File: README.md [hunk 4]
```diff
@@ -1,3 +1,15 @@
 # Signadot Examples
 
 This repository contains examples for Signadot users.
+
```
Comment (foxish) [inline README.md:4]: Can you move this into the specific readmes for the examples to make it self-contained? A copy of the same thing in each example directory's readme.

**Review by foxish — APPROVED** [review]
LGTM, thanks!

cc @jmsktm

**Comment by jmsktm** [comment]
Small observation. The link for `kubectl` isn't showing properly.

image

**Comment by Freezerburn** [comment]
@jmsktm Good catch! Fixed.

**Pull request merged**